Machine.Specifications.ReSharper.Provider.10/RunStrategies/MspecOutOfProcessUnitTestRunStrategy.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using JetBrains.ReSharper.TaskRunnerFramework;
+using JetBrains.ReSharper.UnitTestFramework;
 using JetBrains.ReSharper.UnitTestFramework.Strategy;
 using Machine.Specifications.ReSharperRunner;
 
 namespace Machine.Specifications.ReSharperProvider.RunStrategies
 {
-    public class MspecOutOfProcessUnitTestRunStrategy : OutOfProcessUnitTestRunStrategy
+    public class MspecOutOfProcessUnitTestRunStrategy : TaskRunnerOutOfProcessUnitTestRunStrategy
```

Comment:
Add `[SolutionComponent]` attribute to this class, then we can inject it into `MspecServiceProvider`.